lib/WebpackError.js

```diff
@@ -0,0 +1,21 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Jarid Margolin @jaridmargolin
+*/
+"use strict";
+
+module.exports = class WebpackError extends Error {
+	constructor() {
+		super();
+
+		if(this.initialize) {
+			this.initialize.apply(this, arguments);
```

Comment:
btw. you don't need the `if(Error.hasOwnProperty("captureStackTrace"))` because `captureStackTrace` is available in every node.js we support. It was not available in older node.js versions.